suse2013/static/css/style.css

```diff
@@ -1724,13 +1724,13 @@ p, .simplelist td {
 }
 
 .title .xref, .subtitle .xref, .table-title .xref, .example-title .xref,
-.figure-title .xref, .procedure-title .xref, .orderedlist-title .xref,
+.figure-title .xref, .procedure-title .xref, .task-title .xref, .orderedlist-title .xref,
 .itemizedlist-title .xref, .variablelist-title .xref, .qandadiv-title .xref,
 .title .link, .subtitle .link, .table-title .link, .example-title .link,
-.figure-title .link, .procedure-title .link, .orderedlist-title .link,
+.figure-title .link, .procedure-title .link, .task-title .link, .orderedlist-title .link,
 .itemizedlist-title .link, .variablelist-title .link, .qandadiv-title .link,
 .title .ulink, .subtitle .ulink, .table-title .ulink, .example-title .ulink,
-.figure-title .ulink, .procedure-title .ulink, .orderedlist-title .ulink,
+.figure-title .ulink, .procedure-title .ulink, .task-title .ulink, .orderedlist-title .ulink,
```

Comment:
oh my. i can't even look at this css. totally not your fault but mine. but i almost feel like opening a bug about generating fewer of those class names.

to be clear: your changes are fine and should work. this is more of a note-to-past-self.

Reply:
Ok. 👍